src/ProblemDetailsResponseFactory.php

```diff
@@ -4,6 +4,8 @@
  * @see       https://github.com/mezzio/mezzio-problem-details for the canonical source repository
  * @copyright https://github.com/mezzio/mezzio-problem-details/blob/master/COPYRIGHT.md
  * @license   https://github.com/mezzio/mezzio-problem-details/blob/master/LICENSE.md New BSD License
+ *
+ * phpcs:disable WebimpressCodingStandard.Namespaces.UnusedUseStatement.UnusedUse
```

Comment:
Reported in webimpress/coding-standard#118

JSON_PRETTY_PRINT is not detected as a used contants.

Reply:
@michalbundyra this is where the issue is.

Reply:
Yeah, I know ;-) I have a fix already, will release tomorrow, thanks 👍

Reply:
@geerteltink it has been resolved now in webimpress/coding-standard - 1.1.6 release.

Reply:
I saw your new release. I'll update this PR with the change soon.